loadtests/loadtest.py

```diff
@@ -0,0 +1,21 @@
+from os import getenv
+from molotov import scenario
+
+
+SERVER_URL = getenv('SERVER_URL', 'https://fx-breach-alerts.herokuapp.com')
+HOMEPAGE_URL = SERVER_URL
+SCAN_URL = '{}/scan'.format(SERVER_URL)
+# This hash is for test@example.com (via `node scripts/make-sha1-hashes.js`)
+TEST_EMAIL_HASH = '567159d622ffbb50b11b0efd307be358624a26ee'
```

Comment:
@nhnt11 Added a comment and made the SERVER_URL a bit more configurable via an env variable.